src/vs/editor/common/model/textModel.ts

```diff
@@ -2253,6 +2254,7 @@ export class ModelDecorationOptions implements model.IModelDecorationOptions {
 
 	private constructor(options: model.IModelDecorationOptions) {
 		this.description = options.description;
+		this.blockClassName = options.blockClassName || null;
```

Comment:
You could use `cleanClassName` to avoid that bad stuff (e.g. `"<>'` are passed in this string).